caffe2/core/operator_c10wrapper.h

```diff
@@ -116,14 +116,13 @@ class C10OperatorWrapper final : public Operator<Context> {
   }
 
   void pushOutputParameters_() {
+    std::vector<at::Tensor> preallocated_outputs;
+    preallocated_outputs.reserve(num_output_parameters);
```

Comment:
I'm shocked to see this is a property. It needs a trailing undescore..

Reply:
This diff is about changing the c10->caffe2 wrapper and this file is caffe2->c10, i.e. the wrapper for other direction. In this diff, I only do the minimal changes to the caffe2->c10 wrapper needed to keep it working, but there's another diff rewriting the caffe2->c10 wrapper and fixing this issue on the way, see https://github.com/pytorch/pytorch/pull/17781.